jobs/pull-kubernetes-e2e-gce-cri.sh

```diff
@@ -50,7 +50,7 @@ export GINKGO_PARALLEL="y"
 # This list should match the list in kubernetes-e2e-gce.
 export GINKGO_TEST_ARGS='--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]'
 export FAIL_ON_GCP_RESOURCE_LEAK="false"
-export PROJECT="kubernetes-pr-cri-validation"
+export PROJECT="${PROJECT:-kubernetes-pr-cri-validation}"
```

Comment:
For now, since `docker -v` will tramp over `--env-file` so if you set it locally it will overwrite the value here, thus it's less an issue.